group_vars/all

```diff
@@ -3,6 +3,7 @@ python_path: /usr/bin/python3.6
 ansible_python_interpreter: /usr/bin/python3
 
 system_group: workers
+user_group: staff
```

Comment:
Update name of this variable to `staff_group`. It reads better for knowing who is in that group (who would be in a `user` group? Everybody?)